dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphTest.java

```diff
@@ -58,12 +62,25 @@ public void setUp() {
     path5.add(foo);
     path5.add(bat1);
     path5.add(baz1); // 2 paths to baz1
+    path6.add(foo);
+    path6.add(bat1);
     
     graph.addPath(path1);
     graph.addPath(path2);
     graph.addPath(path3);
     graph.addPath(path4);
     graph.addPath(path5);
+    graph.addPath(path6);
+  }
+  
+  @Test
+  public void testGetHighestVersionMap() {
+    Map<String, String> map = graph.getHighestVersionMap();
+    Assert.assertEquals("2", map.get("com.google:baz"));
```

Comment:
Again, this reads more straightforwardly to me. assertThat(map).containsExactly(...) seems particularly challenging to follow since it doesn't distinguish between keys and values except by position.